### DISCUSSION TITLE
[Feature Request]: Option to save filter selection in session

### DISCUSSION THREAD

**Discussion by lrljoe** [rappasoft/laravel-livewire-tables#1489] [Feature Requests]
### Overview

Ability to Enable/Disable the storing of Filter Selections in the Session

### Detailed explanation

Should be able to enable on a per-table basis.
Should store the selected filter value(s), and restore them on returning to the table
Should not impact other table(s)

**Comment by lrljoe** [comment]
Rather than saving it in the session, I think a smoother way would be to serialize the array, encrypt, and pass it into an Alpine method that stores it in the browser.

Could then add a "Saved Tables" dropdown really simply.

**Comment by qwasyx0** [comment]
Any update on this? I am willing to sponsor some more if this gets priority, I sent you message on discord @lrljoe

**Reply by lrljoe** [reply]
[https://github.com/rappasoft/laravel-livewire-tables/pull/1910](https://github.com/rappasoft/laravel-livewire-tables/pull/1910)

I need to write some more tests, but I'll validate that the default behaviour has no impact this weekend, and merge into a release.

**Reply by lrljoe** [reply]
Released in latest version
